samples/subsys/ipc/ipm_mcux/remote/src/main_remote.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2018, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <device.h>
+#include <ipm.h>
+
+struct device *ipm;
+
+void ping_ipm_callback(void *context, u32_t id, volatile void *data)
+{
+	ipm_send(ipm, 1, 0, (const void *)data, 4);
+}
+
+
+
+void main(void)
+{
+	ipm = device_get_binding("MAILBOX_0");
```

Comment:
Use MAILBOX_0_LABEL instead of hardcoding the string

Reply:
fixed